packages/calcite-components/src/components/date-picker/date-picker.e2e.ts

```diff
@@ -28,6 +28,108 @@ describe("calcite-date-picker", () => {
+        shadowSelector: "calcite-date-picker-month",
+        targetProp: "--calcite-date-picker-month-day-corner-radius",
+      },
+      "--calcite-date-picker-day-range-background-color-hover": {
+        targetProp: "--calcite-date-picker-month-day-range-background-color-hovered",
+        shadowSelector: "calcite-date-picker-month",
+      },
+      "--calcite-date-picker-day-range-background-color-selected": {
+        targetProp: "--calcite-date-picker-month-day-range-background-color-selected",
+        shadowSelector: "calcite-date-picker-month",
+      },
+      "--calcite-date-picker-day-text-color-hover": {
+        targetProp: "--calcite-date-picker-month-day-text-color-hover",
+        shadowSelector: "calcite-date-picker-month",
+      },
+      "--calcite-date-picker-day-text-color": {
+        shadowSelector: "calcite-date-picker-month",
+        targetProp: "--calcite-date-picker-month-day-text-color",
+      },
+      "--calcite-date-picker-day-text-selected": {
+        shadowSelector: "calcite-date-picker-month",
+        targetProp: "--calcite-date-picker-month-day-text-selected",
+      },
+      "--calcite-date-picker-header-background-color": {
+        shadowSelector: "calcite-date-picker-month-header",
+        targetProp: "--calcite-date-picker-month-header-background-color",
+      },
+      "--calcite-date-picker-header-button-background-color-active": {
+        targetProp: "--calcite-date-picker-month-header-button-background-color-active",
+        shadowSelector: "calcite-date-picker-month-header",
+      },
+      "--calcite-date-picker-header-button-background-color-hover": {
+        targetProp: "--calcite-date-picker-month-header-button-background-color-hover",
+        shadowSelector: "calcite-date-picker-month-header",
+      },
+      "--calcite-date-picker-header-button-background-color": {
+        targetProp: "--calcite-date-picker-month-header-button-background-color",
+        shadowSelector: "calcite-date-picker-month-header",
+      },
+      "--calcite-date-picker-header-button-text-color-hover": {
+        targetProp: "--calcite-date-picker-month-header-button-text-color-hover",
+        shadowSelector: "calcite-date-picker-month-header",
+      },
+      "--calcite-date-picker-header-button-text-color": {
+        targetProp: "--calcite-date-picker-month-header-button-text-color",
+        shadowSelector: "calcite-date-picker-month-header",
+      },
+      "--calcite-date-picker-header-text-color": {
+        targetProp: "--calcite-date-picker-month-header-text-color",
+        shadowSelector: "calcite-date-picker-month-header",
+      },
+      "--calcite-date-picker-week-header-text-color": {
+        targetProp: "--calcite-date-picker-month-week-header-text-color",
+        shadowSelector: "calcite-date-picker-month",
+      },
+    } as const;
+    themed(
+      `<calcite-date-picker-month-header active-date="${new Date()}" locale-data"${{
```

Comment:
Can you use the [`html` tagged template function](https://github.com/Esri/calcite-components/blob/14036c1354419222bc5b839af8b68675b17ebf62/packages/calcite-components/support/formatting.ts#L35-L37) to help Prettier format embedded HTML ([example](https://github.com/Esri/calcite-design-system/blob/f1ff5e1acd0e802f6c5b635a77dedf8138d7e753/packages/calcite-components/src/components/card/card.e2e.ts#L27-L29))?